universe/Ship.cpp

```diff
@@ -134,7 +134,7 @@ void Ship::Copy(TemporaryPtr<const UniverseObject> copied_object, int empire_id)
             this->m_design_id =             copied_ship->m_design_id;
             for (PartMeterMap::const_iterator it = copied_ship->m_part_meters.begin();
                  it != copied_ship->m_part_meters.end(); ++it)
-            { this->m_part_meters[it->first]; }
+            { this->m_part_meters = copied_ship->m_part_meters; }
```

Comment:
I'm not particularly happy with that solution either, but I can't think of another/better fix. The problem is, I don't know to what extend the AI uses this info for threat assessment, and if we don't fix this visibility issue, a human can gain a significant advantage here, because the weapon part name *is* shown. Meaning, in theory, if you are willing to browse through the list of enemy ships you could calculate the firepower of an enemy fleet. Horrendous micromanagy of course, so that needs to be fixed anyway.

What I don't get why the original code worked for unpaired meters. If I read that code correctly, it just inserts and initializes the part meter, but leaves it at default values and does not set it to the values of the corresponding part meter of the copied ship...?